plugin.video.massengeschmack/.gitignore

```diff
@@ -0,0 +1,27 @@
+# OS-specific files
```

Comment:
But they will still be pulled and be present on the github repository for no reason. Same for specific IDE files. What makes it so hard for you to remove them from the branch you PR from?